falcon/typing.py

```diff
@@ -62,3 +65,30 @@
 Headers = Dict[str, str]
 HeaderList = Union[Headers, List[Tuple[str, str]]]
 ResponseStatus = Union[http.HTTPStatus, str, int]
+
```

Comment:
I swear the above didn't work for me with sinks... But maybe I was testing on an older Mypy version back then, or made some other mistake :thinking: